store/sqlstore/team_store.go

```diff
@@ -440,8 +440,8 @@ func (s SqlTeamStore) AnalyticsTeamCount() store.StoreChannel {
 var TEAM_MEMBERS_WITH_SCHEME_SELECT_QUERY = `
 	SELECT
 		TeamMembers.*,
-		TeamScheme.DefaultChannelUserRole TeamSchemeDefaultUserRole,
-		TeamScheme.DefaultChannelAdminRole TeamSchemeDefaultAdminRole
+		TeamScheme.DefaultTeamUserRole TeamSchemeDefaultUserRole,
+		TeamScheme.DefaultTeamAdminRole TeamSchemeDefaultAdminRole
```

Comment:
Yeah glad you caught that too... that was part of what [I was investigating yesterday](https://github.com/mattermost/mattermost-server/compare/MM-10757#diff-2bf0f4f1c7df80cba60bcc6435a86849R445) too.